apps/api/src/persistence/database/database.provider.ts

```diff
@@ -10,7 +10,6 @@ import { ArangoDatabaseForCollection } from './arango-database-for-collection';
 import { IDatabaseProvider } from './interfaces/database-provider';
 import { ArangoCollectionID } from './types/ArangoCollectionId';
 
-// TODO Should we rename this `ArangoDatabaseProvider` ?
```

Comment:
I still need to run through the app flow with you to be able to comment on this, but it does seem reasonable to make the name change if this is only providing the connection with arango and not any other type of db like postgres